### PR TITLE
Login feature

### PR DESCRIPTION
**Pull request by Ozleblebici**
All local test passed, ready to review, Thanks guys

### CODE REVIEW

**Comment by BushraChelik** [comment]
Hi Talha, I'm checking this PR, will inform you soon

**Comment by BushraChelik** [comment]
Could you also send a screen shot of your local test result?

**Comment by Ozleblebici** [comment]
Sure here is the test results 
image

File: src/LoginTests/FinalTest.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package LoginTests;
+
+public class FinalTest {
+//    /dsasdsadsadasdasdsad
```
Comment (BushraChelik) [inline src/LoginTests/FinalTest.java]: Hi @Ozleblebici it will be good to use  long rather then int data type

File: src/TEST1/MainBranch.java [hunk 2]
```diff
@@ -1,5 +1,6 @@
 package TEST1;
 
 public class MainBranch {
-    // created this file in main branch !!!
```
Comment (BushraChelik) [inline src/TEST1/MainBranch.java:4]: @Ozleblebici Why did you change this line ?

Comment (Ozleblebici) [reply src/TEST1/MainBranch.java:4]: hi @BushraChelik I need to update to complete finalTest file. IT is all good.

File: src/LoginTests/FinalTest.java (hunk 1 above)
Comment (Ozleblebici) [reply src/LoginTests/FinalTest.java]: hi @BushraChelik I updated with [dc8efc7] commit, ready for review again, thanks

Comment (BushraChelik) [reply src/LoginTests/FinalTest.java]: looks goood thanks

**Review by BushraChelik — APPROVED** [review]
ALL GOOD FOR ME
